documentation/smg/10_how_to_use_the_SAN_area.md

```diff
@@ -0,0 +1,35 @@
+# How to use the SAN area
+
+ISIS has provided a storage area in which PACE can keep large test data. The
+area should contain data files from which we derive smaller sets of data
+(alongside scripts recording how the smaller sets were generated) and files
+that are too large to store in git repositories.
+
+A FedID is required in order to mount the SAN area. In order to mount the SAN
+on PACE's systems, a dedicated user ID with which to mount the SAN area has
+been provided to PACE.
+
+## Mounting the SAN on Jenkins
+
+To mount the SAN when running a Jenkins pipeline, you should use the dedicated
+user ID provided to PACE. The credentials for this are stored in a credentials
+file with ID `SAN_credentials_file`.
+
+One way to mount the drive on Linux is to use `gio`. The code fragment below
+gives an example of how to copy the file `README.txt` from the SAN area in a
+Jenkinsfile. ANVIL requires the use of `dbus-run-session` when mounting the
+drive.
+
+```groovy
+stage('SAN-Mount-And-Copy') {
+  steps {
+    withCredentials([file(credentialsId: 'SAN_credentials_file', variable: 'san_credentials')]) {
+      sh '''
+        echo "gio --version; gio mount smb://isis.cclrc.ac.uk/Shares/PACE_Project_Tool_Source/ < ${san_credentials}; gio copy smb://isis.cclrc.ac.uk/Shares/PACE_Project_Tool_Source/README.txt . -p" > mount_script.sh
```

Comment:
Should the share location be a Jenkinsfile parameter? (stored in the Jenkins job configuration or keystore)? 
- it's used twice in this mini script, probably more times in real code
- it could change
- it probably shouldn't be stored in GitHub